projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss

```diff
@@ -7,10 +7,69 @@
     display: flex;
     align-items: center;
     flex: 0 0 65px;
-    flex-basis: 48px;
-    background: var(--theme-background-color);
+    flex-basis: 96px;
+    background: var(--theme-page-layout-header-background-color);
     border-bottom: 1px solid var(--theme-border-color, rgba(0, 0, 0, 0.07));
     padding: 0 24px;
+
+      .adf-breadcrumb-item {
+        font-size: 20px !important;
+        font-weight: 400 !important;
+        letter-spacing: 0.15px !important;
+      }
+
+      .app-search-input {
+        background: none;
+      }
+    
+      .action-bar {
+        display: flex;
+        flex: auto;
+        height: 32px;
+        margin-left: 24px;
+      }
+    
+      .aca-mat-button {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        letter-spacing: 0.25px;
+        font-weight: 500;
+        font-size: 14px;
+        font-style: normal;
+        border-radius: 6px;
+        border: none;
+      }   
+    
+      .aca-create-button {
+        width: 71px !important;
+        min-width: 71px;
+        height: 32px;
+        background-color: var(--theme-create-button-background-color);
+        color: var(--theme-create-button-text-color);
+        text-overflow: ellipsis;
+        display: flex;
+        padding: 0;
+      }
+    
+      .aca-upload-button {
+        width: 73px !important;
+        min-width: 73px;
+        height: 32px;
+        background-color: var(--theme-upload-button-background-color);
+        color: var(--theme-upload-button-text-color);
+        text-overflow: ellipsis;
+        display: flex;
+        padding: 0;
+        margin-left: 12px;
+      }
+    
+      .action-bar>adf-toolbar-divider {
```

Comment:
you have action bar already defined above (line 25), can you move this part there?